### DISCUSSION TITLE
Email is invalid, taken or not associated with any account!

### DISCUSSION THREAD

**Discussion by rezahedi** [community/community#38496] [Other Feature Feedback, Questions, & Ideas] [answered]
I have a Github personal account with my personal email, I created a repository and in my local git set my company email as git config user.email and then pushed my commits to my remote github repository. Now I can not add my company email to my profile, or create new Github account, It shows this email is invalid or already taken! so I tried to do a reset password, but it shows that the email address is invalid or not associated with any account!!

I removed all the repositories too, but still can not use my company email address.

Thank you for any help,

**Comment by airtower-luna** [comment]
That sounds like something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help with what email address is or isn't taken. Good luck!

**Reply by rezahedi** [reply]
Thanks, Actually I did submit ticket to GitHub support few weeks ago, but didn't receive any feedback, so I shared here maybe someone faced the same issue like me too.

**Comment by david-engelmann** [comment]
From your settings, you should be able to navigate to the email settings section, (see the screenshot below).

![image](https://user-images.githubusercontent.com/81196676/200386708-7c1d074d-2086-47e0-830c-a6e8ae1e0631.png)

Once in that section you should be able to attach additional emails including your work email, (see the screenshot below).

![image](https://user-images.githubusercontent.com/81196676/200386875-cc347dbc-dd09-4da1-bee7-2364c8111fd0.png)

If it still is saying the email has already been used to create an account, try going through [the password reset flow](https://github.com/password_reset) to investigate further. If you still can't add the email and you can't access an account associated with the email, then reach out to support@github.com, they can help resolve the issue from there!

**Reply by rezahedi** [reply]
Thanks David, I did try all of them, but it keeps show me email invalid, taken or not associated with any accounts!

**Reply by david-engelmann** [reply]
check out my comments below.

**Comment by david-engelmann** [comment, marked as answer]
If the support@github.com is taking a long time to resolve, you can try to go through the account section found in [the support dashboard](https://support.github.com/request). [You can also reach out on their twitter](https://twitter.com/github), they are pretty responsive!

**Comment by david-engelmann** [comment]
It sounds like you'll need assistance from github support. It seems like you've used a few different emails in the past. If you want to see what email was used for a commit, just add .patch to the end of the commit url! It doesn't look like the commit you are referring to was in a public repository. You can also try to use the github api to find a username associated with the email you are trying to use, [this stackoverflow post walks through the call to make](https://stackoverflow.com/questions/44888187/get-github-username-through-primary-email)!

**Reply by rezahedi** [reply]
Thanks David for the help, I tried the github api, with both my personal and company emails, it shows one result for my personal but zero for my company email.
I submitted another ticket to the support, It looks like no other way except waiting for Github support's feedback.

**Reply by david-engelmann** [reply]
I'd try to go through the account section found in [the support dashboard](https://support.github.com/request). [I'd also reach out to them on their twitter](https://twitter.com/github)! Good luck friend!

**Comment by rezahedi** [comment]
I contacted support@github.com for my problem, and they responded after a while and solved my problem.

Thanks everyone for the helps.

**Reply by adamc00** [reply]
Was the solution to follow the [Unlinking your email address from a locked account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account) procedure from GitHub.com documentation?

**Reply by rezahedi** [reply]
I'm not sure, people from GitHub support fixed it.
